sequencer/src/persistence/sql.rs

```diff
@@ -2285,11 +2302,12 @@ impl MembershipPersistence for Persistence {
         epoch: EpochNumber,
         stake: ValidatorMap,
         block_reward: Option<RewardAmount>,
+        stake_table_hash: Option<StakeTableStateHash>,
     ) -> anyhow::Result<()> {
         let mut tx = self.db.write().await?;
 
-        let stake_table_bytes =
-            bincode::serialize(&(stake, block_reward)).context("serializing stake table")?;
+        let stake_table_bytes = bincode::serialize(&(stake, block_reward, stake_table_hash))
```

Comment:
I think we should just store stake table bytes into stake column. We were already doing it wrong for SQL because it was serializing (stake table, block_reward) and deserializing it into (stake table)